common/src/main/java/org/opensearch/ml/common/transport/model_group/MLModelGroupGetRequest.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.model_group;
+
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+import lombok.experimental.FieldDefaults;
+import org.opensearch.action.ActionRequest;
+import org.opensearch.action.ActionRequestValidationException;
+import org.opensearch.core.common.io.stream.InputStreamStreamInput;
+import org.opensearch.core.common.io.stream.OutputStreamStreamOutput;
+import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.common.io.stream.StreamOutput;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+
+import static org.opensearch.action.ValidateActions.addValidationError;
+
+@Getter
+@FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+@ToString
+public class MLModelGroupGetRequest extends ActionRequest {
+
+    String modelGroupId;
+    boolean returnContent;
+
+    @Builder
+    public MLModelGroupGetRequest(String modelGroupId, boolean returnContent) {
+        this.modelGroupId = modelGroupId;
+        this.returnContent = returnContent;
```

Comment:
Why add this parameter? Can you add some example request/response to the description part?

Reply:
We might not need this. This is mostly related to model content. Removing this for model group API